src/Sylius/Behat/Resources/config/suites/ui/checkout/addressing.yml

```diff
@@ -0,0 +1,21 @@
+# This file is part of the Sylius package.
+# (c) Paweł Jędrzejewski
+
+default:
+    suites:
+        ui_checkout_addressing:
```

Comment:
Not sure if this should be a separate suite for each step. Maybe just ``checkout`` suite?